parts/chapters/subsections/8.3/DIFFCGAS.fodt

```diff
@@ -4351,16 +4351,18 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
     </table:table>
     <text:h text:style-name="P18462" text:outline-level="4"><text:bookmark-start text:name="__RefHeading___Toc10367_37011683881954 Copy 1"/>Description<text:bookmark-end text:name="__RefHeading___Toc10367_37011683881954 Copy 1"/></text:h>
-    <text:p text:style-name="P12729">The DIFFCGAS keyword defines the gas diffusion coefficients for each compositional component in the model and for each PVT region, for when the Molecular Diffusion option has been activated by the DIFFUSE keyword in the RUNSPEC section. This keyword is optional as OPM Flow will automatically calculate the coefficients, as described by Sandve et al.<text:note text:id="ftn183" text:note-class="footnote"><text:note-citation>183</text:note-citation><text:note-body>
-        <text:p text:style-name="Footnote">Tor Harald Sandve1, Sarah E. Gasda, Atgeirr Rasmussen, <text:span text:style-name="T7193">and</text:span> Alf Birger Rustad. Convective dissolution in field scale CO<text:span text:style-name="T4092">2</text:span> storage simulation using the OPM Flow simulator. Submitted to TCCS 11 – Trondheim Conference on CO<text:span text:style-name="T4092">2</text:span> Capture, Transport and Storage Trondheim, Norway – June 21-23, 2021.</text:p></text:note-body></text:note>, if the DIFFCGAS keyword is absent from the input deck. <text:s/>The keyword thus allows one to overwrite the automatically calculated values.</text:p>
-    <text:p text:style-name="_40_TextBody">The keyword should only be used if the CO2STORE keyword and either the GASWAT or the GAS and WATER keywords in the RUNSPEC section, have also be activated for the gas-water two component model. </text:p>
+    <text:p text:style-name="_40_TextBody">The DIFFCGAS keyword defines the gas diffusion coefficients assuming the standard mole fraction formulation for each compositional component in the model and for each PVT region, for when the Molecular Diffusion option has been activated by the DIFFUSE keyword in the RUNSPEC section. </text:p>
+    <text:p text:style-name="P12729">This keyword is optional as OPM Flow will automatically calculate the coefficients assuming the mole fraction formulation, as described by Sandve et al.<text:note text:id="ftn183" text:note-class="footnote"><text:note-citation>183</text:note-citation><text:note-body>
+        <text:p text:style-name="Footnote">Tor Harald Sandve1, Sarah E. Gasda, Atgeirr Rasmussen, <text:span text:style-name="T7193">and</text:span> Alf Birger Rustad. Convective dissolution in field scale CO<text:span text:style-name="T4092">2</text:span> storage simulation using the OPM Flow simulator. Submitted to TCCS 11 – Trondheim Conference on CO<text:span text:style-name="T4092">2</text:span> Capture, Transport and Storage Trondheim, Norway – June 21-23, 2021.</text:p></text:note-body></text:note>, if the DIFFAGAS and DIFFCGAS keywords are absent from the input deck. <text:s/>The keyword thus allows one to overwrite the automatically calculated values.</text:p>
```

Comment:
```suggestion
        <text:p text:style-name="Footnote">Tor Harald Sandve, Sarah E. Gasda, Atgeirr Rasmussen, <text:span text:style-name="T7193">and</text:span> Alf Birger Rustad. Convective dissolution in field scale CO<text:span text:style-name="T4092">2</text:span> storage simulation using the OPM Flow simulator. Submitted to TCCS 11 – Trondheim Conference on CO<text:span text:style-name="T4092">2</text:span> Capture, Transport and Storage Trondheim, Norway – June 21-23, 2021.</text:p></text:note-body></text:note>, if the DIFFAGAS and DIFFCGAS keywords are absent from the input deck. <text:s/>The keyword thus allows one to overwrite the automatically calculated values.</text:p>
```
Not sure why the 1 was there originally. (note sure they footnote shows with 1%)